snekmer/scripts/motif_rescore.py

```diff
@@ -0,0 +1,137 @@
+"""
+Created on Tue Apr 25 15:14:39 2023
+
+author: @tnitka
+"""
+
+# ---------------------------------------------------------
+# Imports
+# ---------------------------------------------------------
+import pickle
+# from datetime import datetime
+
+import snekmer as skm
+import pandas as pd
+import numpy as np
+import gzip
+import gc
+# from typing import Any, Dict, List, Optional
```

Comment:
Comment cleanup

Reply:
This and the other formatting issues are now addressed in 4ff1b0a1ab5bc3cf70d370fbf9e697db0c17959c.